sdk/dotnet/Codepipeline/Pipeline.cs

```diff
@@ -27,11 +27,11 @@ public partial class Pipeline : Pulumi.CustomResource
         public Output<string> Arn { get; private set; } = null!;
 
         /// <summary>
-        /// An artifact_store block. Artifact stores are documented below.
+        /// One or more artifact_store blocks. Artifact stores are documented below.
         /// * `stage` (Minimum of at least two `stage` blocks is required) A stage block. Stages are documented below.
         /// </summary>
-        [Output("artifactStore")]
-        public Output<Outputs.PipelineArtifactStore> ArtifactStore { get; private set; } = null!;
+        [Output("artifactStores")]
+        public Output<ImmutableArray<Outputs.PipelineArtifactStores>> ArtifactStores { get; private set; } = null!;
```

Comment:
It looks like more breaking changes related to pluralization?

Reply:
uuuggggh :/